recommenders/content_based.py

```diff
@@ -1,132 +1,9 @@
-from recommenders import data_movies, default_poster_url
-from sklearn.feature_extraction.text import TfidfVectorizer
-from sklearn.metrics.pairwise import linear_kernel
-import pandas as pd
-import numpy as np
-from fuzzywuzzy import fuzz
-# create a function to find the closest title
-def matching_score(a,b):
-   return fuzz.ratio(a,b)
```

Comment:
You need to import the linear_kernel function from the sklearn.metrics.pairwise module before using it.